public/editor-client/src/defaultTemplates/utils.ts

```diff
@@ -1,387 +1,81 @@
-        mobileFontSizeSuffix: "px",
-        mobileFontWeight: 700,
-        mobileLineHeight: 1.4,
-        mobileLetterSpacing: 0
-      },
-      {
-        deletable: "off",
-        id: "heading5",
-        title: "Heading 5",
-        fontFamily: "overpass",
-        fontFamilyType: "google",
-        fontSize: 20,
-        fontSizeSuffix: "px",
-        fontWeight: 700,
-        lineHeight: 1.6,
-        letterSpacing: 0,
-        tabletFontSize: 17,
-        tabletFontSizeSuffix: "px",
-        tabletFontWeight: 700,
-        tabletLineHeight: 1.7,
-        tabletLetterSpacing: 0,
-        mobileFontSize: 17,
-        mobileFontSizeSuffix: "px",
-        mobileFontWeight: 700,
-        mobileLineHeight: 1.8,
-        mobileLetterSpacing: 0
-      },
-      {
-        deletable: "off",
-        id: "heading6",
-        title: "Heading 6",
-        fontFamily: "overpass",
-        fontFamilyType: "google",
-        fontSize: 16,
-        fontSizeSuffix: "px",
-        fontWeight: 700,
-        lineHeight: 1.5,
-        letterSpacing: 0,
-        tabletFontSize: 16,
-        tabletFontSizeSuffix: "px",
-        tabletFontWeight: 700,
-        tabletLineHeight: 1.5,
-        tabletLetterSpacing: 0,
-        mobileFontSize: 16,
-        mobileFontSizeSuffix: "px",
-        mobileFontWeight: 700,
-        mobileLineHeight: 1.5,
-        mobileLetterSpacing: 0
-      },
-      {
-        deletable: "off",
-        id: "button",
-        title: "Button",
-        fontFamily: "overpass",
-        fontFamilyType: "google",
-        fontSize: 15,
-        fontSizeSuffix: "px",
-        fontWeight: 700,
-        lineHeight: 1.6,
-        letterSpacing: 0,
-        tabletFontSize: 17,
-        tabletFontSizeSuffix: "px",
-        tabletFontWeight: 700,
-        tabletLineHeight: 1.6,
-        tabletLetterSpacing: 0,
-        mobileFontSize: 15,
-        mobileFontSizeSuffix: "px",
-        mobileFontWeight: 700,
-        mobileLineHeight: 1.6,
-        mobileLetterSpacing: 0
-      }
-    ],
-    id: "bahcadtpvdhdphmhymrsgrwobyzhxcdzytyx",
-    title: "Overpass"
-  }
-];
-
-export const fetchAllElements = async <T>(
-  url: string,
-  id: string,
-  itemsPerPage: number
-): Promise<T[]> => {
-  let allElements: T[] = [];
-
-  const firstPageResponse = await fetch(
-    `${url}?project_id=${id}&per_page=${itemsPerPage}`
-  ).then((r) => r.json());
-
-  const lastPage = firstPageResponse.paginationInfo.lastPage;
-
-  allElements = allElements.concat(firstPageResponse.collections);
-
-  for (let currentPage = 2; currentPage <= lastPage; currentPage++) {
-    const nextPageResponse = await fetch(
-      `${url}?project_id=${id}&per_page=${itemsPerPage}&page_number=${currentPage}`
-    ).then((r) => r.json());
-
-    allElements = allElements.concat(nextPageResponse.collections);
-  }
-
-  return allElements;
-};
+export function convertToCategories(obj: Record<string, string>): Categories[] {
+  return Object.entries(obj).map(([_, title]) => ({
```

Comment:
use `Object.value(obj)` instead of `Object.entries(obj)` to void using of `_` for skip an variable